### ISSUE TITLE
Skype API; feature has been deprecated

### ISSUE THREAD

**Issue by findel** [ShyykoSerhiy/skyweb#59]
Getting this error starting today;

```
node_modules/skyweb/dist/src/utils.js:9
        throw message;
        ^
Failed to get auth requests.null/{"statusCode":404,"body":"{\"status\":{\"code\":40444,\"text\":\"Feature has been deprecated\"}}","headers":{"server":"nginx/1.7.8","date":"Thu, 16 Feb 2017 00:02:14 GMT","content-type":"application/json; ver=1.0; charset=utf-8","content-length":"62","connection":"close","x-stratus-processing-time":"0.0247","x-skype-request-id":"0963ce31","x-stratus-request-id":"0963ce31","x-stratus-deprecated":"Switch to http://aka.ms/contactsservice","pragma":"no-cache","cache-control":"no-store, no-cache, must-revalidate","expires":"Thu, 01 Jan 1970 00:00:01 +0000","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","x-xss-protection":"1; mode=block","strict-transport-security":"max-age=31536000; includeSubdomains;"},"request":{"uri":{"protocol":"https:","slashes":true,"auth":null,"host":"api.skype.com","port":443,"hostname":"api.skype.com","hash":null,"search":null,"query":null,"pathname":"/users/self/contacts/auth-request","path":"/users/self/contacts/auth-request","href":"https://api.skype.com/users/self/contacts/auth-request"},"method":"GET","headers":{"X-Skypetoken":"PRIVATE-REMOVED","cookie":"SC=CC=:CCY=:LC=en:LIM=:TM=1487203211:TS=1487203211:TZ=:VAT=:VER=; skplet=1487289611"}}}
```

**Comment by ShyykoSerhiy** [comment]
Looks like smth has been broken on Microsoft side "Feature has been deprecated."  I'll try to fix this now.

**Comment by ShyykoSerhiy** [comment]
Can you please provide more info? I've just tried to login and everything is fine. When exactly does this error show?

**Comment by ShyykoSerhiy** [comment]
This might occur if Microsoft changed license agreement for live(skype) account. Login via browser may help in this situation.

**Comment by findel** [comment]
Yeah it doesn't happen immediately after log in. It takes a while.

I've just tried a browser log in (via mobile, not at computer). So I'll see what happens.

**Comment by findel** [comment]
Still doing it, sorry.

**Comment by ShyykoSerhiy** [comment]
Yeah, Microsoft has changed "add contact" request endpoint. I'll fix it today.

**Comment by ShyykoSerhiy** [comment]
Ok, seems like they've moved from http polling to web sockets for getting auth requests. I'll remove AuthRequest.ts from next release.

**Comment by ShyykoSerhiy** [comment]
"Fixed" in 0.3.0. 
and on/un were added 
```ts
Skyweb = require('skyweb');
var skyweb = new Skyweb();
const errorListener = (eventName: string, error: string) => {
    console.log(`${errorCount} : Error occured : ${error}`);
    errorCount++;
    if (errorCount === 10) {
        console.log(`Removing error listener`);
        skyweb.un('error', errorListener); // Removing error listener
    }
};
skyweb.on('error', errorListener); //Adding error listener
```

**Comment by findel** [comment]
Neat, thanks!
